src/vm/gdbjit.cpp

```diff
@@ -1785,48 +1827,23 @@ static int getNextPrologueIndex(int from, T &arr, int n)
     return -1;
 }
 
+static NewArrayHolder<WCHAR> WszModuleNames;
```

Comment:
Could you please change the naming convention here to use `g_` prefix too?

Reply:
done